.github/workflows/ci.yml

```diff
@@ -14,9 +14,10 @@ jobs:
     runs-on: ubuntu-22.04
     container:
       image: heroku/heroku:${{ matrix.stack_number }}-build
+      options: --user root
     strategy:
       matrix:
-        stack_number: ["20", "22"]
+        stack_number: ["20", "22", "24"]
```

Comment:
Whilst approving/merging #1248 I noticed that the new Heroku-24 test jobs in the matrix weren't marked as required for branch protection (along with the "Test Metrics (14.10.0)" job too). I've added them to the required list now:
https://github.com/heroku/heroku-buildpack-nodejs/settings/branch_protection_rules/17110526